docs/requirements.txt

```diff
@@ -2,3 +2,4 @@ pyenchant
 Sphinx>=4.5.0
 sphinxcontrib-spelling
 blacken-docs
+sphinx-copybutton
```

Comment:
Thanks for the response, and yeah most probably there is, I will look into it.

Reply:
What’s your concern @smithdc1? Anecdotally we’ve been using this same dependency for the Wagtail docs for two years now, haven’t had a problem yet.

Reply:
@arthurvasconcelos did you see the [comment](https://github.com/django/django/pull/16342#issuecomment-1728734503) on the last PR that this conflicts with [JavaScript in djangoproject.com](https://github.com/django/djangoproject.com/blob/main/djangoproject/static/js/mod/clippify.js) and perhaps should be solved there rather than adding a dependency here?